src/lib.rs

```diff
@@ -1924,6 +1924,51 @@ macro_rules! transmute_ref {
     }}
 }
 
+/// Includes a file and safely transmutes it to an arbitrary type.
+///
+/// The file will be included as a byte array, `[u8; N]`, which will be
+/// transmuted to another type, `T`. `T` is inferred from the calling context,
+/// and must implement [`FromBytes`].
+///
+/// The file is located relative to the current file (similarly to how modules
+/// are found). The provided path is interpreted in a platform-specific way at
+/// compile time. So, for instance, an invocation with a Windows path containing
+/// backslashes `\` would not compile correctly on Unix.
+///
+/// # Examples
+///
+/// Assume there are two files in the same directory with the following
+/// contents:
+///
+/// File 'data' (no trailing newline):
+///
+/// ```text
+/// abcd
+/// ```
+///
+/// File 'main.rs':
+///
+/// ```rust
+/// use zerocopy::include_bytes;
+/// # macro_rules! include_bytes {
+/// # ($file:expr) => { zerocopy::include_bytes!(concat!("../testdata/include_bytes/", $file)) };
+/// # }
+///
+/// fn main() {
+///     let as_u32: u32 = include_bytes!("data");
+///     assert_eq!(as_u32, u32::from_ne_bytes([b'a', b'b', b'c', b'd']));
+///     let as_i32: i32 = include_bytes!("data");
+///     assert_eq!(as_i32, i32::from_ne_bytes([b'a', b'b', b'c', b'd']));
+/// }
+/// ```
+#[macro_export]
+macro_rules! include_bytes {
+    ($file:expr $(,)?) => {
+        // TODO: Is this hygienic?
+        $crate::transmute!(*core::include_bytes!($file))
```

Comment:
Following the advice of [this guide](https://gist.github.com/Kestrer/8c05ebd4e0e9347eb05f265dfb7252e1):
```suggestion
        $crate::transmute!(*::core::include_bytes!($file))
```

Reply:
Done.